runtime/vm/visible.c

```diff
@@ -142,3 +155,55 @@ checkVisibility(J9VMThread *currentThread, J9Class* sourceClass, J9Class* destCl
 
 	return result;
 }
+
+#if defined(J9VM_OPT_VALHALLA_NESTMATES)
+static VMINLINE UDATA loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, BOOLEAN isSourceClass) {
```

Comment:
This should only be called once per class in non-failing cases as the `nestHost` will either be itself or filled in by the first call.  I don't think it needs to be VMINLINEd.

For classes without a nest attribute, we can set the `nestHost` to themselves in createramclass.cpp and never call this function.  (TODO: confirm we do that already)